LessonSix/src/test/java/TestArrayOneAndFour.java

```diff
@@ -0,0 +1,40 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Parameterized.class)
+public class TestArrayOneAndFour {
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][]{
+                {new int[]{1, 1, 4, 4, 4, 1, 4, 1}, true},
+                {new int[]{9, 6, 3, 5, 9, 3, 5, 1}, false},
+                {new int[]{1, 4, 8, 7}, false}
+        });
+    }
+    private int[] in;
+    private boolean out;
+
+    public TestArrayOneAndFour(int[] in, boolean out) {
+        this.in = in;
+        this.out = out;
+    }
+
+    private ArrayOneAndFour array;
+
+    @Before
+    public void startTest() {
+        array = new ArrayOneAndFour();
+    }
+
+    @Test
+    public void testOnly1And4() {
+        Assert.assertEquals(ArrayOneAndFour.Only1And4(in), out);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.